go-controller/pkg/metrics/ovn.go

```diff
@@ -380,4 +433,6 @@ func RegisterOvnControllerMetrics() {
 	go ovnControllerConfigurationMetricsUpdater()
 	// ovn-controller coverage show metrics updater
 	go coverageShowMetricsUpdater(ovnController)
+	// ovn-controller subnet configuration metrics updater
+	go ovnControllerSubnetMetricsUpdater()
```

Comment:
Why is this executed in its own go-routine? It's not needed in this case.

Reply:
Do we expect this to be run periodically? So that if the users do change/add CIDRs we will keep reflecting the values appropriately?

Reply:
I am not sure. I was thinking this happens once during stat up. If that behavior needs to change we can do that as well

Reply:
> Why is this executed in its own go-routine? It's not needed in this case.

All metrics updates are happening in separate go routines. Just wanted to be consistent.

Reply:
> Do we expect this to be run periodically? So that if the users do change/add CIDRs we will keep reflecting the values appropriately?

It shouldn't. Even if a user changes the CIDR ranges for these values, they will need to restart ovnkube for the changes to take affect, which means we will just pick up the values when we restart. 

> Just wanted to be consistent.

No need for that here.

Reply:
Done